lib/KCheckbox.vue

```diff
@@ -228,4 +231,13 @@
     }
   }
 
+  // KIcon overrides
+  .indeterminate,
+  .unchecked,
+  .checked {
+    top: 0;
+    width: $checkbox-height;
+    height: $checkbox-height;
+  }
```

Comment:
I'm a bit confused why there are three classes defined rather than just a single `checkbox` class?

Reply:
@indirectlylit honestly because I saw the same convention used in KRadioButton's template. Changed them now to `checkbox-icon`